### PR TITLE
DevKit updates for 3.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by OskarStark** [comment]
@franmomu can you please check why the build is now failing? Thanks

File: tests/custom_bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+$application = new Application(new AppKernel());
+$application->setAutoExit(false);
+
+// Load fixtures of the AppTestBundle
+$input = new ArrayInput([
+    'command' => 'doctrine:mongodb:fixtures:load',
+    '--no-interaction' => false,
+]);
+$application->run($input, new ConsoleOutput());
```
Comment (OskarStark) [inline tests/custom_bootstrap.php:28]: Why do we use console output here and nulloutput down below?

Comment (franmomu) [reply tests/custom_bootstrap.php]: good catch! I probably left it when doing some debugging.

**Comment by OskarStark** [comment]
@franmomu Psalm is failing 🤔

**Comment by franmomu** [comment]
> @franmomu Psalm is failing 🤔

Looks like it's fine now

**Pull request merged**